src/main/java/kitchenpos/domain/OrderTables.java

```diff
@@ -0,0 +1,33 @@
+package kitchenpos.domain;
+
+import java.util.List;
+import org.springframework.util.CollectionUtils;
+
+public class OrderTables {
+
+    public static final int NOT_MANY_TABLE_SIZE = 2;
```

Comment:
```suggestion
    private static final int NOT_MANY_TABLE_SIZE = 2;
```
해당 클래스 내에서만 사용되어서 접근 제어자를 Private으로 해도 될 것 같습니당